### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gholadr**
updating readme so that it references the new website URL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Move And Whap (MaW)
 
-A game demo showcasing how to build a simple, multiplayer game where all user actions are on-chain. The game is playable and running [here](https://move-and-whap.netlify.app/), with plans to implement the game on a subnet.
+A game demo showcasing how to build a simple, multiplayer game where all user actions are on-chain. The game is playable and running [on our MaWnet subnet](https://mnw.room40.network:3001//).
```
Comment (trileeeee) [inline README.md]: The double slash at the end of the link seems to be redundant. Does it serve a purpose?

Comment (gholadr) [reply README.md]: oh its wrong, should be no dash, you're right

Comment (gholadr) [reply README.md]: fixed it

**Review by trileeeee — APPROVED** [review]
Okay nice, LGTM now

**Pull request merged**